crates/apub/src/protocol/objects/page.rs

```diff
@@ -111,6 +117,17 @@ impl Attachment {
       Attachment::Document(d) => d.url,
     }
   }
+
+  pub(crate) fn alt_text(self) -> Option<String> {
```

Comment:
Check this to make sure its okay. Tests are passing, but it seems strange to use `name` for all these types.